omigami/data_models.py

```diff
@@ -0,0 +1,109 @@
+# TODO: rename as data_models
+
+from typing import Iterable, List, Dict, Union
+from dataclasses import dataclass
+from omigami.data_types import NumpyArray
+from omigami.model import Estimator
+
+
+@dataclass
+class Split:
+    id: int
+    train_indices: NumpyArray
+    test_indices: NumpyArray
+
+
+@dataclass
+class InputData:
```

Comment:
This is a bit of a very generic name don't you think? Maybe call it Dataset -  still generic but a bit less^^

Reply:
`InputDataset` :]

Reply:
Does it matter that it is Input? Why Input lol^^